.github/workflows/main.yaml

```
@@ -52,3 +52,26 @@ jobs:
       - name: Check source code licenses
         run: |
           docker run --platform=linux/amd64 -v ${PWD}:/src ghcr.io/google/addlicense -v -check -l BSD-2-Clause -c "ChipFlow" -s=only -ignore **/__init__.py **/*.py
+          
+  chipflow-silicon-submit:
```

```suggestion
  silicon-submit:
```

(or `chipflow-submit`; I prefer shorter names for checks since they can be unwieldy in the GitHub UI.)

---

Added this